.github/workflows/ci-workflow.yml

```diff
@@ -0,0 +1,69 @@
+name: "CI Tests"
+on:
+  pull_request:
+    types: [assigned, opened, synchronize, reopened]
```

Comment:
should we enable for only `opened` 🤔 I think we no need to run the test again on `assigned or reopened`

Reply:
I have updated the to `on: pull_request`, IMO it's still worth to re-run on `reopen`. Without types, it runs on `opened, synchronize, reopened` by default. Reference: https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request